# Description: teach gsd about the gnome-calculator snap .desktop name
# Ubuntu: https://launchpad.net/bugs/1796499
# Upstream: needs a better solution for snaps
Index: gnome-settings-daemon-3.30.1.2/plugins/media-keys/gsd-media-keys-manager.c
===================================================================
--- gnome-settings-daemon-3.30.1.2.orig/plugins/media-keys/gsd-media-keys-manager.c
+++ gnome-settings-daemon-3.30.1.2/plugins/media-keys/gsd-media-keys-manager.c
@@ -2516,7 +2516,7 @@ do_action (GsdMediaKeysManager *manager,
                 do_media_action (manager, timestamp);
                 break;
         case CALCULATOR_KEY:
-                do_execute_desktop_or_desktop (manager, "org.gnome.Calculator.desktop", "gnome-calculator.desktop", timestamp);
+                do_execute_desktop_or_desktop (manager, "org.gnome.Calculator.desktop", "gnome-calculator_gnome-calculator.desktop", timestamp);
                 break;
         case CONTROL_CENTER_KEY:
                 do_execute_desktop_or_desktop (manager, "gnome-control-center.desktop", NULL, timestamp);
